src/views/Projects/index.jsx

```diff
@@ -2,6 +2,7 @@ import { Component, Fragment } from 'react';
 import { withStyles } from 'material-ui/styles';
```

Comment:
A rebase is needed after https://github.com/mozilla-frontend-infra/bug-issue-ui/pull/22 is squashed and merged. By rebasing, this file will not be included in this PR anymore.